### PR TITLE
Add 'How to build driver with phantomjs-prebuilt'

### PR DESCRIPTION
**Pull request by vtange**
- [x] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

Added Instructions for something like this in the comments above "Find Executable" function of PhantomJS
In response to https://github.com/SeleniumHQ/selenium/issues/2069

### CODE REVIEW

**Comment by jleyba** [comment]
You've updated the docs on a private function. If you want this notice to be included in the generated API docs, you should add a fileoverview comment to the top of the file. For example, see [selenium-webdriver/chrome.js](https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/chrome.js#L18) (here are the [public docs generated](http://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/chrome.html) extracted from that file).

**Comment by vtange** [comment]
OK. Moved the docs to the top of the file under @fileoverview. Ring me up if it still needs work

File: javascript/node/selenium-webdriver/phantomjs.js [hunk 1]
```diff
@@ -15,6 +15,28 @@
 // specific language governing permissions and limitations
 // under the License.
 
+/**
+ * @fileoverview Defines a {@linkplain Driver WebDriver} client for the PhantomJS
+ * web browser. By default, it is expected that PhantomJS exists in your working directory.
```
Comment (jleyba) [inline javascript/node/selenium-webdriver/phantomjs.js]: By default, it is expected that the PhantomJS executable can be located on your
    [PATH](https://en.wikipedia.org/wiki/PATH_(variable))

File: javascript/node/selenium-webdriver/phantomjs.js [hunk 2]
```diff
@@ -15,6 +15,28 @@
...
+ * 
+ *  __Using a Custom PhantomJS Binary__
+ * 
+ * If you have PhantomJS.exe placed somewhere other than the root of your working directory,
+ * you can build a custom Capability and attach the executable's location to the Capability
+ * 
+ * For example, if you're using the phantomjs-prebuilt module (https://www.npmjs.com/package/phantomjs-prebuilt) from npm:
```
Comment (jleyba) [inline javascript/node/selenium-webdriver/phantomjs.js]: For example, if you're using the [phantomjs-prebuilt](https://www.npmjs.com/package/phantomjs-prebuilt)
    module from npm:

File: javascript/node/selenium-webdriver/phantomjs.js [hunk 3]
```diff
@@ -15,6 +15,28 @@
...
+ * 
+ * //setup custom phantomJS capability
```
Comment (jleyba) [inline javascript/node/selenium-webdriver/phantomjs.js]: Indent each line in this example by at least 4 spaces so it renders as a code block:

    *    // setup a custom phantomJS capability
    *    var phantomjs_exe = require('phantomjs-prebuilt').path;

**Pull request merged**